### ISSUE TITLE
level hierarchy incorrect when implement multi parents

### ISSUE THREAD

**Issue by Farahkh** [BALKANGraph/OrgChartJS#428]
hi, I am trying to implement multiple parents in my family tree using this example:
https://jsfiddle.net/gh/get/library/pure/BALKANGraph/orgchartjs-demos/tree/master/CustomTemplatePartner
but when I do so my tree hierarchy gets missed up, can you please guide me on how to fix it?

before implementing the example the tree looks like this:
![image](https://user-images.githubusercontent.com/1657931/90360597-30065580-e06d-11ea-9271-e1bf16064651.png)

after implementing the example it looks like this:
![image](https://user-images.githubusercontent.com/1657931/90360653-55935f00-e06d-11ea-8843-ee7b9193b257.png)

**Comment by ZornitsaSerbezova** [comment]
Hi Farah,
There was a bug in the [demo ](https://balkangraph.com/OrgChartJS-Demos/CustomTemplatePartner)that we have fixed. Actually, you just need to change the name of the tag in the configuration as well in the data. The problem was that the demo was mixing up with the main implementation of a [partner ](https://balkangraph.com/OrgChartJS-Demos/Partner)that we did later, because of the tag name.
